src/Donation/DonationRequest.php

```diff
@@ -10,11 +10,23 @@
 use Misd\PhoneNumberBundle\Validator\Constraints\PhoneNumber as AssertPhoneNumber;
 use Ramsey\Uuid\Uuid;
 use Ramsey\Uuid\UuidInterface;
+use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;
 use Symfony\Component\Validator\Constraints as Assert;
 
+/**
+ * TODO: add endSubscriptionAt in the list when end subscription will be merged.
+ *
+ * @UniqueEntity(
+ *     service="AppBundle\Validator\UniqueDonationSubscriptionValidator",
+ *     message="donation.subscription.not_unique",
+ *     fields={
+ *     "emailAddress",
+ *     "duration"
```

Comment:
why duration? I think you need juste email to validate this donation request

Reply:
Because we need want unique subscription

Reply:
yes, right :)